Python/Product/AnalysisBrowser/FunctionView.cs

```diff
@@ -37,29 +38,45 @@ public FunctionView(IModuleContext context, string name, IPythonFunction member,
 
         public string OverloadSummary {
             get {
-                var args = new List<HashSet<string>>();
-
-                if (_summary == null) {
-                    foreach (var overload in _function.Overloads) {
-                        var parameters = overload.GetParameters();
-                        for (int i = 0; i < parameters.Length; ++i) {
-                            while (args.Count <= i) {
-                                args.Add(new HashSet<string>());
-                            }
+                if (_overloadSummary != null) {
+                    return _overloadSummary;
+                }
 
-                            args[i].Add(parameters[i].Name);
+                var args = new List<HashSet<string>>();
+                var types = new List<HashSet<string>>();
+
+                foreach (var overload in _function.Overloads) {
+                    var parameters = overload.GetParameters();
+                    for (int i = 0; i < parameters.Length; ++i) {
+                        while (args.Count <= i) {
```

Comment:
Depends how many overloads we have, how many parameters each has, and what order we get to them in.